### PR TITLE
bumped dependency intellij-common to 0.0.1 (#342)

### LINKED ISSUE #342: master doesn't compile
```gradle clean buildPlugin``` produces the following compilation error:
```
/Users/andredietisheim/Documents/jboss-workspaces/intellij-tekton/src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java:492: error: cannot find symbol
            ExecHelper.executeWithUI(envVars, outputToEditor(fileName),command, "pipelinerun", "logs", pipelineRun, "-f", "-n", namespace);
                      ^
  symbol:   method executeWithUI(Map<String,String>,Consumer<String>,String,String,String,String,String,String,String)
  location: class ExecHelper
```

### PR DESCRIPTION
**Pull request by adietish**
fixes #342

### CODE REVIEW

**Comment by jeffmaury** [comment]
I don't see the rationale for this as our PRs are ok. At least 0.0.1 should be used.

File: build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ intellij {
 dependencies {
     compile 'io.fabric8:openshift-client:5.0.0'
     compile 'io.fabric8:tekton-client:5.0.0'
-    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1-SNAPSHOT'
+    compile 'com.redhat.devtools.intellij:intellij-common:0.0.2-SNAPSHOT'
```
Comment (jeffmaury) [inline build.gradle:32]:
```suggestion
    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1'
```

**Comment by adietish** [comment]
@jeffmaury I'm a bit astonished that 0.0.1 has the API change while 0.0.1-SNAPSHOT hasn't. Shouldn't 0.0.2-SNAPSHOT have it?

**Comment by adietish** [comment]
@jeffmaury turns out that my comment above is the exact reason for the error: my local cache had a version of 0.0.1-SNAPSHOT without the change and thus tekton failed to compile for me. Compiling and publishing latest intellij-common to the local repo of course didn't help since it had been bumped.

**Comment by jeffmaury** [comment]
Yes that's an issue on your env as our builds are OK but fine to use 0.0.1 released

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**